src/views/AddItem.jsx

```diff
@@ -1,3 +1,66 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+const defaultItem = { itemName: '', daysUntilNextPurchase: 7 };
+export function AddItem({ listToken }) {
+	const [item, setItem] = useState(defaultItem);
+	const [status, setStatus] = useState('');
+
+	const updateItem = (e) => {
+		if (status) setStatus('');
+		if (
+			e.target.name !== 'daysUntilNextPurchase' &&
```

Comment:
This double condition didn't work as expected. Because the "soon" is a default value in the form field, this input always has a value. So if I don't put in anything for the item, I can still submit an empty item to the database. 

Maybe change the form field so that the default has no actual value, a placeholder like "select frequency"

Reply:
This is interesting because we can see from future wireframe that 'Soon' or one of the timeframes should be default set (https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/add-item.png). 

Another option might be checking if item name is an empty string, if default frequency must be chosen.

Reply:
@xynree / @yuff1006 / @h-yung nice catch looking ahead and keeping things in scope, you'll definitely be able to tackle this in issue 9! One thing to consider when issue 9 pops up is using Optional Chaining!